server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -9,6 +9,7 @@ const sinon = require('sinon')
 const { factories, helpers } = require('brave-alert-lib')
 const { braveAlerter, db, server } = require('../../../index')
 const { locationDBFactory, sensorsVitalDBFactory } = require('../../../testingHelpers')
+// const { sendLowBatteryAlert } = require('../../../vitals')
```

Comment:
Is there a reason that this line is only commented out, as opposed to removed?

Reply:
Oh whoops thank you for catching that! That line shouldn't be in there. I thought I would maybe need it when I was writing the test but I didn't and I guess forgot to delete it. Thanks!!